ui/apps/pmm/src/contexts/grafana/grafana.provider.tsx

```diff
@@ -1,45 +1,195 @@
 import { FC, PropsWithChildren, useEffect, useRef, useState } from 'react';
-import { GrafanaContext } from './grafana.context';
 import { useLocation, useNavigate, useNavigationType } from 'react-router';
+import { GrafanaContext } from './grafana.context';
 import {
   GRAFANA_SUB_PATH,
   PMM_NEW_NAV_GRAFANA_PATH,
   PMM_NEW_NAV_PATH,
 } from 'lib/constants';
 import { DocumentTitleUpdateMessage, LocationChangeMessage } from '@pmm/shared';
-import messenger from 'lib/messenger';
 import { getLocationUrl } from './grafana.utils';
 import { updateDocumentTitle } from 'lib/utils/document.utils';
-import { useColorMode } from 'hooks/theme';
 import { useKioskMode } from 'hooks/utils/useKioskMode';
+import { useColorMode } from 'hooks/theme';
+import { useSetTheme } from 'themes/setTheme';
+
+type Mode = 'light' | 'dark';
```

Comment:
What's the reason for declaring a `Mode` type when there's `ColorMode` in `ui/packages/shared/src/types.ts`?